### PR TITLE
Feat : added Homepage

### PR DESCRIPTION
**Pull request by Priya-Patel31**
The homepage is divided into various components which are as follows  :
* HeroSection
* Features 
* Categories 
* Products 
* Brands

### CODE REVIEW

File: src/pages/home/components/brands/brands.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
+import { Fragment } from "react";
+
+const Brands = () => {
+  return (
+    <Fragment>
```
Comment (JantuDeb) [inline src/pages/home/components/brands/brands.js]: You can use `<> </>` instead of `<Fragment>`

File: src/shared/components/header/header.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+import {FaBars} from "react-icons/fa"
+
+const Header = () => {
+  return (
+    <nav className="nav-container">
+      <div className="left-navbar">
+        <div className="hamburger-icon">
+          <FaBars />
+        </div>
+        <Logo></Logo>
```
Comment (JantuDeb) [inline src/shared/components/header/header.js:15]: If you are using this loge here then it's not necessary to create a component with 2-3 lines of html.

File: src/shared/components/nav-icons/nav-icons.js [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+              <FaUserAlt className="text-xs icon-size"/>
+            </span>
+            <span className="nav-list-link">User</span>
+          </div>
+          <div className="dropdown-content">
+            <a href="/authentication/signin/signin.html" className="text-xs">
+              Login
+            </a>
+            <a href="/profile_page/profile_page.html" className="text-xs">
+              Profile
```
Comment (JantuDeb) [inline src/shared/components/nav-icons/nav-icons.js:19]: I think one link will be there either. If logged in then `Profile` otherwise `Login`

**Review by JantuDeb — COMMENTED** [review]
Suggested some changes, another thing is that you can add a static product array and use it in state so you can make your components accordingly.

**Review by Raghav888 — APPROVED** [review]
Looks good.

File: src/pages/home/components/brands/brands.js (hunk 1 above)
Comment (Priya-Patel31) [reply src/pages/home/components/brands/brands.js]: Done !!

File: src/shared/components/header/header.js (hunk 2 above)
Comment (Priya-Patel31) [reply src/shared/components/header/header.js:15]: But I want to break it in smaller components .

File: src/shared/components/nav-icons/nav-icons.js (hunk 3 above)
Comment (Priya-Patel31) [reply src/shared/components/nav-icons/nav-icons.js:19]: Right now I am converting into JSX later I will create a PR for the functionality purpose that time I will make these changes .

**Review by JantuDeb — APPROVED** [review]

**Pull request merged**